arangod/RocksDBEngine/RocksDBCollection.h

```diff
@@ -141,6 +141,11 @@ class RocksDBCollection final : public RocksDBMetaCollection {
                 RevisionId expectedRev, ManagedDocumentResult& previous,
                 OperationOptions& options);
 
+  Result performUpdateOrReplace(transaction::Methods* trx,
+                                velocypack::Slice newSlice,
```

Comment:
I think `newSlice` could be `const` because apparently none of its non-`const` methods are used in this function.